src/website/src/pages/ec/utilities/html-tag/docs/usage.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Two things, this page should be added also in `src/website/public/admin/config.yml` for the cms to handle it and since we have now the EU sections in the website we should also add one there `src/website/src/pages/eu/utilities/html-tag`, finally also this usage page in EU should be added in the cms configuration :)

Reply:
I am thinking that we will handle this page on our side, so i'd say that the inclusion in the cms is only optional, no real need for that...